### ISSUE TITLE
loss function details

### ISSUE THREAD

**Issue by aliabavi** [joonson/syncnet_python#7]
Hi,

Where did you define the loss? 
What is an appropriate value for margin?

Thanks

**Comment by joonson** [comment]
The training script is not included in this repository, but contrastive loss was used to train the model. We used a margin of 20, but using a different value should not affect the performance.
